Soma-Technologies-Inc/visitAfrica-backend
# #175494070 Return Trip

[start:175494070]
[finish:175494070]

#### What does this PR do?

- user should be able to return trip

Description of Task to be completed?

- **This is my end point**

- POST /api/trip/return_trip

How should this be manually tested?

- Open your terminal 
- Clone this repo by running git clone https://github.com/Soma-Technologies-Inc/visitAfrica-backend
- Run git checkout ft-return-trip-175494070 to navigate to my branch
- Run git pull to make sure that you have my codes in you vs code
- Run npm install for the installation of all needed packages

Screenshots
![return](https://user-images.githubusercontent.com/73258718/102684044-ec1f8a80-41dd-11eb-8c1e-f6b96cd63cee.PNG)


Questions:
Relevant Pivot tracker Id
#175494070
